at_client/lib/src/manager/monitor.dart

```diff
@@ -252,6 +252,7 @@ class Monitor {
     }
     _logger.finer(
         'Authenticating the monitor connection: from result:$fromResponse');
+    if(_preference.privateKey == null) throw Exception('Private key is null in preference'); // JEREMY
```

Comment:
@JeremyTubongbanua please remove `// JEREMY` ... git will tell us who did the commit 😀

Reply:
See f070f6a8de74c0368e89b9f238615a1b8751a364 👍